src/types/hash.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * KeccakHash is a 256 bit hash string encoded as hex with a '0x' at the start
+ *
+ * @type KeccakHash
+ */
+export type KeccakHash = string;
```

Comment:
If I understand how this is supposed to work correctly, I think we should follow putting this into type declarations i.e put them in a .d.ts file and do:
```
declare type KeccakHash = string
```

Reply:
Fixed